app/controllers/admin/users_controller.rb

```diff
@@ -3,7 +3,22 @@ class UsersController < ApplicationController
     before_filter :authorize_admin!
 
     def index
-      @users = User.where(admin: 'f').page(params[:page])
+      @users = User.all.page(params[:page])
+    end
+
+    def edit
+      @user = User.find(params[:id])
+    end
+
+    def update
+      @user = User.find(params[:id])
+      if @user.update_attributes(admin: 't')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.